noxfile.py

```diff
@@ -1,17 +1,26 @@
 import nox
 
+TARGETS = ["meteoblue_dataset_sdk", "tests", "noxfile.py"]
 
-@nox.session
+
+@nox.session(reuse_venv=True)
 def lint(session):
-    lint_tools = ["flake8"]  # "black", "isort",
-    targets = ["meteoblue_dataset_sdk", "tests", "noxfile.py"]
+    lint_tools = ["flake8", "isort"]
+    session.install(*lint_tools)
+    session.run("isort", *TARGETS)
+    session.run("flake8", *TARGETS)
+
+
+@nox.session(reuse_venv=True)
+def lint_dev(session):
+    lint_tools = ["flake8", "black", "isort"]
     session.install(*lint_tools)
-    session.run("flake8", *targets)
-    # session.run("black", "--diff", "--check", *targets)
-    # session.run("isort", "--check-only", *targets)
+    session.run("isort", "-f", *TARGETS)
+    session.run("black", *TARGETS)
+    session.run("flake8", *TARGETS)
 
 
-@nox.session
+@nox.session(reuse_venv=True)
 def tests(session):
```

Comment:
Would it be possible to add on the "github job confiugration file" python versions 3.8,3.9 as well? 
Then nox could be running against all of them and we would ensure the version compatibility

Reply:
I will create a issue ticket for that to keep track. Last time, 3.9 had some issues in CI